public/javascripts/buttons.js

```diff
@@ -0,0 +1,30 @@
+//this is mithril component for buttons in the song clicked page
+var buttonsComponent = (function() {
```

Comment:
name this something more descriptive, and rename the file to something more descriptive (as described in the comment above even).